app/uk/gov/hmrc/apiplatformmicroservice/apidefinition/models/APIDefinition.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.apiplatformmicroservice.apidefinition.models
+
+import uk.gov.hmrc.apiplatformmicroservice.apidefinition.models.APICategory.APICategory
+import uk.gov.hmrc.apiplatformmicroservice.apidefinition.models.APIStatus.APIStatus
+import uk.gov.hmrc.apiplatformmicroservice.apidefinition.models.HttpMethod.HttpMethod
+
+case class APIDefinition(serviceName: String,
+                         name: String,
+                         description: String,
+                         context: String,
+                         requiresTrust: Option[Boolean],
+                         isTestSupport: Option[Boolean] = None,
+                         versions: Seq[APIVersion],
+                         categories: Option[Seq[APICategory]] = None)
+
+object APICategory extends Enumeration {
```

Comment:
I thought we didn't use Scalas Enumeration and instead preferred Enumeratum?
https://github.com/lloydmeta/enumeratum

Reply:
Hmm.  One to discuss

Reply:
Well either Enumeratum or a sealed trait adt would be better than the Enumeration that comes with Scala.